testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestExternalHiveTable.java

```diff
@@ -149,6 +149,38 @@ public void testCreateExternalTableWithInaccessibleSchemaLocation()
         String table = "test_create_external";
         String tableLocation = "/tmp/" + table;
         query(format("CREATE TABLE %s.%s.%s WITH (external_location = '%s') AS SELECT * FROM tpch.tiny.nation", HIVE_CATALOG_WITH_EXTERNAL_WRITES, schema, table, tableLocation));
+
+        String otherSchema = "schema_with_unknown_location";
+        query(format("CREATE SCHEMA %s.%s", HIVE_CATALOG_WITH_EXTERNAL_WRITES, otherSchema));
+
+        String otherTable = "table_with_unknown_schema_location";
+        String otherTableLocation = "/tmp/" + otherTable;
+        query(format("CREATE TABLE %s.%s.%s WITH (external_location = '%s') AS SELECT * FROM tpch.tiny.nation", HIVE_CATALOG_WITH_EXTERNAL_WRITES, otherSchema, otherTable, otherTableLocation));
```

Comment:
here, looks like the schema directory would still exist. can we just create this table in `schema` used for previous testcase ?

Reply:
Correct. IIUC we only need to perform a CREATE TABLE (not CTAS) here using the `schema_without_location`  schema.